Nef_3/include/CGAL/Nef_3/ID_support_handler.h

```diff
@@ -18,7 +18,8 @@
 
 #include <CGAL/Nef_S2/ID_support_handler.h>
 #include <CGAL/Nef_3/SNC_indexed_items.h>
-#include <CGAL/Unique_hash_map.h>
+#include <boost/container_hash/hash.hpp>
```

Comment:
I think this should be   `boost/functional/hash.hpp`  instead.   Note that we have a [testsuite ](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-5.3-Ic-95/Nef_3/TestReport_afabri_x64_Cygwin-Windows10_MSVC2015-Debug-64bits.gz)that fails.